config/locales/en.yml

```diff
@@ -2685,6 +2685,11 @@ See the %{link} to find out more about %{sitename}'s features and to start using
         in your cart have reduced. Here's what's changed:
       now_out_of_stock: is now out of stock.
       only_n_remainging: "now only has %{num} remaining."
+    shop_variant_no_group_buy:
+      add: "Add"
+      increment: "＋" # U+FF0B Fullwidth Plus Sign
+      decrement: "－" # U+FF0D Fullwidth Hyphen-Minus
```

Comment:
this looks weird, do we need this to be translatable? Shouldnt this be an icon in the app? just asking.

Reply:
Do you mean it looks weird in the code or weird in the browser?

I didn't want to assume that plus and minus signs are the best button labels in all regions of the world. So I kept it translatable. We could introduce a graphic symbol in the app but that wouldn't be translatable, more difficult to change and it would need extra loading.

Reply:
I meant weird in the code.
I see.
But every translator will translate this to the normal + sign which I believe will not look so nice. This one is bigger right?

Reply:
I cant think of a case where someone will want to use another symbol and I dont think using text will work UI-wise, right?

Reply:
Yes, it's bigger and it has the same height as the font itself which makes it easier to style.

You can use text, it will just make the buttons bigger. And if the text for increase and decrease has different lengths then the buttons would have different sizes.

Well, I can include those symbols in the code, no problem. Maybe I'll check with translators first.